storagetransfer/aws_s3_compatible_source_request.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Command-line sample that creates a transfer from an AWS S3-compatible source
+to GCS.
+"""
+
+
+import argparse
+
+# [START storagetransfer_transfer_from_s3_compatible_source]
+from google.cloud import storage_transfer
+
+AuthMethod = storage_transfer.S3CompatibleMetadata.AuthMethod
+NetworkProtocol = storage_transfer.S3CompatibleMetadata.NetworkProtocol
+RequestModel = storage_transfer.S3CompatibleMetadata.RequestModel
+
+
+def transfer_from_S3_compat_to_gcs(
+        project_id: str,
+        description: str,
+        source_agent_pool_name: str,
+        source_bucket_name: str,
+        source_path: str,
+        gcs_sink_bucket: str,
+        gcs_path: str,
+        region: str,
+        endpoint: str,
+        protocol: NetworkProtocol,
+        request_model: RequestModel,
+        auth_method: AuthMethod,):
+    """Creates a transfer from an AWS S3-compatible source to GCS"""
```

Comment:
Looks like formatting came off

```suggestion
def transfer_from_S3_compat_to_gcs(
    project_id: str,
    description: str,
    source_agent_pool_name: str,
    source_bucket_name: str,
    source_path: str,
    gcs_sink_bucket: str,
    gcs_path: str,
    region: str,
    endpoint: str,
    protocol: NetworkProtocol,
    request_model: RequestModel,
    auth_method: AuthMethod,
) -> None:
    """Creates a transfer from an AWS S3-compatible source to GCS"""
```

Reply:
thanks